.github/workflows/cmd-check.yaml

```diff
@@ -39,18 +39,33 @@ jobs:
       - uses: r-lib/actions/setup-tinytex@master
         if: contains(matrix.config.args, 'no-manual') == false
 
+      - name: Query dependencies
+        run: |
+          install.packages('remotes')
+          saveRDS(remotes::dev_package_deps(dependencies = TRUE), "depends.Rds", version = 2)
+        shell: Rscript {0}
```

Comment:
What is the `{0}` mean here?

Reply:
It's just the syntax that says "put the code in the `run:` section here"